### PR TITLE
Add integration tests for account-setup

### LINKED ISSUE #865: Write component tests for payments/account-setup
# Problem

We need to write component tests for the `payments/account-setup` container.

We need to test that it should:

- render the organization name
- render the `payments/funds-recipient` component
- render the `payments/bank-account` component

There's no need to test much deeper than that these components are rendered with at least some subcomponent, since they will rely on passing down the `stripeConnectAccount` model in order to show their statuses.

We should also remove the `payments-contact` info from being rendered in this component for the time being.

### PR DESCRIPTION
**Pull request by begedin**
# What's in this PR?

Adds integration tests for the `account-setup` component. 

There are likely to be some overlaps with #931 when merging this.

## References
Fixes #865

### CODE REVIEW

File: tests/pages/components/payments/funds-recipient/verification-document.js [hunk 1]
```diff
@@ -1,9 +1,23 @@
...
+  // since it's near impossible to simulate selecting a file with a filepicker
+  // this does the "next best" thing and triggers the "upload done" action on the component
+  triggerDocumentSubmitted(context, stripeFileId = 'fil_test') {
+    let id = $(this.scope).attr('id');
+    let component = context.container.lookup('-view-registry:main')[id];
+
+    run(component, () => {
+      component.get('onUploadDone').call(component, stripeFileId);
+    });
+  }
```
Comment (JoshSmith) [inline tests/pages/components/payments/funds-recipient/verification-document.js]: Should we remove the simulation where we use `sinon` (in the photo drop zone test) and replace with this instead? I know that is one of the things breaking in our tests in Ember 2.10.

**Comment by JoshSmith** [comment]
Question before you merge, but otherwise 👍

File: tests/pages/components/payments/funds-recipient/verification-document.js (hunk 1 above)
Comment (begedin) [inline tests/pages/components/payments/funds-recipient/verification-document.js]: We might want to. Can extract into helper.

Comment (begedin) [inline tests/pages/components/payments/funds-recipient/verification-document.js]: It didn't work for me now either, btw

**Pull request merged**